diff --git a/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.cjs b/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.cjs
index 82dafe9..85a18c2 100644
--- a/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.cjs
+++ b/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.cjs
@@ -1292,7 +1292,7 @@ function extraswaps(input) {
 }
 async function embed2(props) {
   var _a, _b;
-  await shell_default(props.shellModule);
+  await shell_default({ module_or_path: props.shellModule });
   const runtime = new ShellRuntime(props.container);
   embed(props.container, runtime, {
     fontFamily: (_a = props.fontFamily) != null ? _a : "monospace",
diff --git a/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.js b/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.js
index a40b662..33be6fe 100644
--- a/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.js
+++ b/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.js
@@ -1282,7 +1282,7 @@
   }
   async function embed2(props) {
     var _a, _b;
-    await shell_default(props.shellModule);
+    await shell_default({ module_or_path: props.shellModule });
     const runtime = new ShellRuntime(props.container);
     embed(props.container, runtime, {
       fontFamily: (_a = props.fontFamily) != null ? _a : "monospace",
diff --git a/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.mjs b/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.mjs
index 944af37..2458adc 100644
--- a/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.mjs
+++ b/node_modules/@duckdb/duckdb-wasm-shell/dist/shell.mjs
@@ -1249,7 +1249,7 @@ function extraswaps(input) {
 }
 async function embed2(props) {
   var _a, _b;
-  await shell_default(props.shellModule);
+  await shell_default({ module_or_path: props.shellModule });
   const runtime = new ShellRuntime(props.container);
   embed(props.container, runtime, {
     fontFamily: (_a = props.fontFamily) != null ? _a : "monospace",
